library/src/layout/par.rs

```diff
@@ -887,7 +916,11 @@ fn linebreak_optimized<'a>(vt: &Vt, p: &'a Preparation<'a>, width: Abs) -> Vec<L
                 // has minimum cost. All breakpoints before this one become
                 // inactive since no line can span above the mandatory break.
                 active = k;
-                MIN_COST + if attempt.justify { ratio.powi(3).abs() } else { 0.0 }
+                if attempt.justify {
```

Comment:
why do we not need `MIN_COST` anymore? and if we don't, the comment should be updated.

Reply:
Because the `active = k` line effectively makes sure that the breakpoint is mandatory.

Comments fixed in a3bed3acf921957afca8c386d010919e5465628a